src/little_dict.jl

```diff
@@ -54,6 +54,8 @@ LittleDict(ks, vs) = LittleDict(collect(ks), collect(vs))
 function LittleDict{K,V}(itr) where {K,V}
     ks = K[]
     vs = V[]
+    # handle the special case of a single Pair
+    itr isa Pair{K, V} && return LittleDict([first(itr)], [last(itr)])	
```

Comment:
Can we make this a top level method?

maybe
`LittleDict{K,V}(single_item::Union{Pair{K,V}, Tuple{K,V}} where {K,V}`

Following the more general pattern of:
Replace `if x isa` with a new method for multiple-dispatch